### PR TITLE
[Nox.Integration] Setup ETL Box #978.

### LINKED ISSUE #978: [Nox.Integration] Setup ETL Box
- relates to #973
- depends on #976
-
Add ETL Box Nuget package without license key, having the free features by defaults

Add extensibility to Nox to allow services to setup their on keys 

(Ask @andresharpe  for the key to test it properly, do not commit any license)

### PR DESCRIPTION
**Pull request by AndreyDegtyarev**
Introduced services to get license key for EtlBox

### CODE REVIEW

File: src/Nox.Integration/Extensions/NoxEtlBoxExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using ETLBoxOffice.LicenseManager;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Nox.Integration.Abstractions;
+
+namespace Nox.Integration.Extensions;
+
+public static class NoxEtlBoxExtensions
+{
+    public static void UseEtlBox(this IApplicationBuilder applicationBuilder)
```
Comment (rochar) [inline src/Nox.Integration/Extensions/NoxEtlBoxExtensions.cs]: and if no license is provided and the user wants to use the free ETL Box?

File: src/Nox.Integration/Nox.Integration.csproj [hunk 2]
```diff
@@ -38,6 +38,10 @@
 
     <ItemGroup>
       <PackageReference Include="ETLBox" Version="3.0.2" />
+      <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```
Comment (rochar) [inline src/Nox.Integration/Nox.Integration.csproj]: Do we need the Http dependency? this could be used in a class Library?

**Review by rochar — CHANGES_REQUESTED** [review]

File: src/Nox.Integration/Nox.Integration.csproj (hunk 2 above)
Comment (AndreyDegtyarev) [reply src/Nox.Integration/Nox.Integration.csproj]: Microsoft.AspNetCore.Http.Abstractions is used to refer on IApplicationBuilder

Comment (rochar) [reply src/Nox.Integration/Nox.Integration.csproj]: Create an Extension to IServiceColletion would be enough? are we using anything that comes from IApplicationBuilder?

**Review by jan-schutte — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**